scripts/state/migrate-from-0.3.py

```diff
@@ -9,8 +9,18 @@
 
 filename = os.path.join(os.path.dirname(os.path.realpath(__file__)), '../../near/res/testnet.json')
 q = json.loads(open(filename).read())
+new_records = []
 
-q['records'] = q['records'][0]
+#q['records'] = q['records'][0]
```

Comment:
delete?

Reply:
We will need this line (I think) when we migrate from current testnet, but the current testnet.json is already migrated so no need to use this line.